### PR TITLE
[TASK] Use own configuration files for tests

### PR DESCRIPTION
**Pull request by lukaszuznanski**
Resolves: #902

### CODE REVIEW

File: Tests/Functional/FunctionalTests.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<phpunit
```
Comment (oliverklee) [inline Tests/Functional/FunctionalTests.xml:1]: AFAIK, the PHPUnit configuration (at least as written by PHPUnit) contains the XML prologue and the schema information:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.6/phpunit.xsd"
```

**Review by oliverklee — CHANGES_REQUESTED** [review]

File: Tests/Unit/UnitTests.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            <!--
+                This path either needs an adaption in extensions, or an extension's
+                test location path needs to be given to phpunit.
+            -->
+            <directory suffix="Test.php">./</directory>
+        </testsuite>
+    </testsuites>
+    <!-- @todo: change tag to 'coverage' when TF requires phpunit > 9 -->
+    <filter>
+        <!-- @todo: change tag to 'include' when TF requires phpunit > 9 -->
```
Comment (oliverklee) [inline Tests/Unit/UnitTests.xml:35]: We're already using PHPUnit 9. So we should witch to `coverage` and `include` here.

**Review by oliverklee — CHANGES_REQUESTED** [review]

**Review by oliverklee — APPROVED** [review]

**Comment by sbuerk** [comment]
@oliverklee @lukaszuznanski 

Little bit late. TBH the bootstraps should also not be used from the the `typo3/testing-framework` vendor folder and copied to the project.

The background is, that there are scenarious where it's needed to make some changes based on the project/extension.

Beside this, the chosen location is kind of flattered. Configuration for all tools should be collected in one place and not distributed throughout a lot of places (and not the root folder)
to have a clean and "one place" to look it up. But that is maybe more a personally flavour - where I highly disagree to take this places as `best practice`.

**Comment by oliverklee** [comment]
Thanks, I've created #1082 for moving these files.

**Pull request merged**